src/benchmarks.zig

```diff
@@ -424,6 +473,21 @@ pub fn benchmarkCSV(
 
                     // print column results
                     try writer_average.print("{s}, {d}, {d}, {d}, {d}\n", .{ arg_name, min, max, mean, std_dev });
+
+                    // collect the metric
+                    if (maybe_metrics.*) |*metrics| {
+                        const fmt_size = std.fmt.count("{s}({s})", .{ def.name, arg_name });
+                        const name_buf = try allocator.alloc(u8, fmt_size);
+                        const name = try std.fmt.bufPrint(name_buf, "{s}({s})", .{ def.name, arg_name });
+
+                        const metric = Metric{
+                            .name = name,
+                            .unit = time_unit.toString(),
+                            .value = mean,
```

Comment:
Question: Any reason why the mean is set directly here, without the processing that is applied in the float case as done in here https://github.com/Syndica/sig/pull/395/files#diff-0fb4beeff922a420b2b5b7caff8f618005553add01a26cd17fe65bf29bb5b213R544

Reply:
in this case the return type is a Duration, which cant be a float so we know that mean will be a u64 and require no conversion -- returning multiple values supports floats which requires the additional work